src/puppetlabs/rbac_client/services/activity.clj

```diff
@@ -53,6 +54,8 @@
           authenticated-connection-limits {:max-connections-per-route (get-in-config [:activity-consumer :max-connections-per-route-auth] 20)
                                            :max-connections-total (get-in-config [:activity-consumer :max-connections-total-auth] 20)}
           client (create-client (merge authenticated-connection-limits ssl-config))]
+      (log/info (i18n/trs "Connection limit per route for authenticated clients has been set to {0}" (:max-connections-per-route authenticated-connection-limits)))
```

Comment:
Would you mind adding something about the name of the service to each of these messages for both services? Since they will both appear in the same logs for every application that uses them, it would be ideal to be able to differentiate.

Reply:
that makes sense :), on it!